package.json

```diff
@@ -47,7 +47,7 @@
     "@nx/web": "19.3.2",
     "@nx/webpack": "19.3.2",
     "@nx/workspace": "19.3.2",
-    "@playwright/experimental-ct-react": "^1.41.2",
+    "@playwright/experimental-ct-react": "^1.49.1",
```

Comment:
had to update the playwright because of failing browsers installation, similar as [here](https://github.com/microsoft/playwright/issues/33051)

Reply:
can we handle these things in separate PR, this is unrelated to the scope of this PR.